starter/src/index.html

```diff
@@ -66,74 +60,10 @@
 	<link href="<%= htmlWebpackPlugin.options.dllFiles.css[i] %>" rel="stylesheet">
 	<% } %>
 	<% } %>
-
-	<!--TODO uncomment this when HtmlElementsPlugin is enabled -->
-	<!--< % if (webpackConfig.htmlElements.headTags) { % >-->
-	<!-- Configured Head Tags  -->
-	<!--< % = webpackConfig.htmlElements.headTags % >-->
-	<!--< % } % >-->
 </head>
-<!-- add the full-curtain class if you want to enable the condensed mode by default -->
-<!-- add the dense-mode class if you want to enable the dense mode by default -->
 
 <body class="">
 
-<!-- FIXME reimplement these functionalities -->
```

Comment:
We should keep those features in mind to re-add them later with the help of Julien :)
-> Separate issue?